src/components/microlearning/VideoDetailPage.jsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect } from 'react';
+import { useParams, Link, useLocation } from 'react-router-dom';
+import {
+  Container, Breadcrumb, Row, MediaQuery, breakpoints, Badge, Skeleton,
+} from '@openedx/paragon';
+import loadable from '@loadable/component';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+import { useEnterpriseCustomer } from '../app/data';
+import { Sidebar } from '../layout';
+import './styles/VideoDetailPage.scss';
+import DelayedFallbackContainer from '../DelayedFallback/DelayedFallbackContainer';
+import { useVideoData } from './data/hooks';
+import { features } from '../../config';
+
+const VideoPlayer = loadable(() => import(/* webpackChunkName: "videojs" */ '../video/VideoPlayer'), {
+  fallback: (
+    <DelayedFallbackContainer>
+      <Skeleton height={200} />
+    </DelayedFallbackContainer>
+  ),
+});
+
+const VideoDetailPage = () => {
+  const { edxVideoID } = useParams();
+  const { data: enterpriseCustomer } = useEnterpriseCustomer();
+  const location = useLocation();
+
+  const [videoData, isLoading, error] = useVideoData(edxVideoID);
+
+  const customOptions = {
+    showPlaybackMenu: true,
+    showTranscripts: true,
+    transcriptUrls: videoData?.transcriptUrls,
+  };
+
+  useEffect(() => {
+    if (videoData?.videoURL) {
+      VideoPlayer.preload();
+    }
+  }, [videoData?.videoURL]);
+
+  const routeLinks = [
+    {
+      label: 'Explore Videos',
+      to: `/${enterpriseCustomer.slug}/videos`,
+    },
+  ];
+  if (location.state?.parentRoute) {
+    routeLinks.push(location.state.parentRoute);
+  }
+
+  if (isLoading) {
+    return (
+      <DelayedFallbackContainer>
+        <Skeleton height={400} />
+      </DelayedFallbackContainer>
+    );
+  }
+
+  // Comprehensive error handling will be implemented upon receiving specific error use cases from the UX team
+  // and corresponding Figma designs.
+  if (error || !features.FEATURE_ENABLE_VIDEO_CATALOG) {
+    return (
+      <div className="text-center py-5">
+        <h1>404</h1>
+        <p>{error?.message}</p>
+      </div>
+    );
+  }
+
+  return (
+    <Container size="lg" className="pt-3">
+      <div className="small">
+        <Breadcrumb
+          links={routeLinks}
+          activeLabel={videoData.courseTitle}
+          linkAs={Link}
+        />
+      </div>
+      <Row>
+        <article className="col-12 col-lg-9">
+          <div className="d-flex flex-column align-items-start flex-grow-1 video-container">
+            <div className="d-flex flex-row align-items-center justify-content-between">
+              <h2 data-testid="video-title" className="mb-0">
+                {videoData.courseTitle}
+              </h2>
+              <span className="small ml-2 mt-2">
+                {videoData.videoDuration && `(${videoData.videoDuration} minutes)`}
```

Comment:
On stage, we don't have durations for some of the videos. So I think you should use optional chaining operator here.